dev-docs/RFCs/proposals/pydeck-module-loading-rfc.md

```diff
@@ -29,7 +29,7 @@ commonjs and AMD are two ways to package up JS modules so that they can be impor
 
 ### Dynamic script loading
 
-loaders.gl has implemented a [dynamic script loading system](https://github.com/uber-web/loaders.gl/tree/master/modules/loader-utils/src/lib/library-utils) to support optional, on-demand loading of large loader libraries and worker threads.
+loaders.gl has implemented a [dynamic script loading system](https://https://github.com/visgl/loaders.gl/tree/master/modules/loader-utils/src/lib/library-utils) to support optional, on-demand loading of large loader libraries and worker threads.
```

Comment:
You have an extra `https://` here, and in at least some other places

Reply:
It looks like the changes from the most recent commit have an extra `https://`

Reply:
Thanks will fix this.